srcpkgs/xidlehook/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'xidlehook'
+pkgname=xidlehook
+version=0.10.0
+revision=1
+build_wrksrc="xidlehook-daemon"
+build_style=cargo
+configure_args="$(vopt_if audio '--features pulse' '--no-default-features')"
+hostmakedepends="pkg-config python3"
+makedepends="libxcb-devel libXScrnSaver-devel $(vopt_if audio pulseaudio-devel)"
+short_desc="Xautolock rewrite in Rust, with a few extra features"
+maintainer="MOIS3Y <stepan@zhukovsky.me>"
+license="MIT"
+homepage="https://gitlab.com/jD91mZM2/xidlehook"
+distfiles="${homepage}/-/archive/${version}/${pkgname}-${version}.tar.gz"
+checksum=66751b78c5174c0e430d9becd749771d593f388db348514d8c3256670dc77cd8
+build_options_default="audio"
+desc_option_audio="Don't invoke the timer when any audio is playing (Only works with Pulseaudio)"
```

Comment:
Randomly named build options are hardly useful, this should probably just use the `pulseaudio` build option.

Reply:
Done too.
After the changes, I re-ran the build and check with the linter
```sh
./xbps-src pkg xidlehook
./xbps-src -a armv6l -Q pkg xidlehook
xlint srcpkgs/xidlehook/template
```